### PR TITLE
ORLP-170. Manage own decks: Implement UI.

### PR DESCRIPTION
**Pull request by b0hdan**
Task is fully completed and this part of the program should work just perfectly. I've done only the ui part of the task, which is to i
mplement managing decks including viewing them, creating new or editing existing ones and deleting them. The server part of this task isn't included in this subtask.

### CODE REVIEW

File: src/app/page/cabinet/userDecks/user.decks.component.html [hunk 1]
```diff
@@ -5,16 +5,124 @@
...
-    <div class="decks-list-item" *ngFor="let deck of decks; let i = index" (click)="onDeckClicked(deck, i)">
-      <span>{{deck.name}}</span>
-      <button class="btn btn-danger btn-sm"
-      style="float: right">-</button>
+  <div class="list-button-container">
+    <div class="decks-list">
+      <div class="decks-list-item" *ngFor="let deck of decks; let i = index" (click)="onDeckClicked(deck, i)">
+        <span>{{deck.name}}</span>
+        <button class="btn btn-danger btn-sm" data-toggle="modal" data-target="#delete-deck"
+                (click)="setSelectedDeck(deck)" style="float: right">-
```
Comment (IhorF) [inline src/app/page/cabinet/userDecks/user.decks.component.html]: It is bad practice to use "style" in this place. Please remove in css file.

Comment (b0hdan) [reply src/app/page/cabinet/userDecks/user.decks.component.html]: Got it) It'll be ready in a moment!

File: src/app/page/cabinet/userDecks/user.decks.component.html [hunk 2]
```diff
@@ -5,16 +5,124 @@
...
   </div>
 
-  <div class="decks-list">
-    <div class="decks-list-item" *ngFor="let deck of decks; let i = index" (click)="onDeckClicked(deck, i)">
-      <span>{{deck.name}}</span>
-      <button class="btn btn-danger btn-sm"
-      style="float: right">-</button>
+  <div class="list-button-container">
+    <div class="decks-list">
+      <div class="decks-list-item" *ngFor="let deck of decks; let i = index" (click)="onDeckClicked(deck, i)">
```
[the remark this answers is not in the document]
Comment (b0hdan) [reply src/app/page/cabinet/userDecks/user.decks.component.html]: Yes, it is necessary.

[the remark this answers is not in the document]
Comment (b0hdan) [reply src/app/page/cabinet/userDecks/user.decks.component.html]: And yes, it's for selecting an HTML element and changing it's property from typescript component.

**Pull request merged**